packages/components/src/spinner/styles.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css, keyframes } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS, CONFIG } from '../utils';
+
+const spinAnimation = keyframes`
+	from {
+		transform: rotate(0deg);
+	}
+	to {
+		transform: rotate(360deg);
+	}
+ `;
+
+export const StyledSpinner = styled.svg`
+	width: ${ CONFIG.spinnerSize }px;
+	height: ${ CONFIG.spinnerSize }px;
+	display: block;
+	margin: auto;
```

Comment:
As I gave one last look at the code changes, I just noticed a couple of differences that I'd like to discuss:

- `display`: it used to be `inline-block`, it's currently `block`. I personally like `block` more, but this change could be potentially breaking for some layouts
- `margin`: it used to be `5px 11px 0`, it's not `auto`. This is also a potentially breaking change. I personally believe that a component should not define rules for its own margin, and so my preference would be to either keep things as they are, or to introduce a breaking change by removing the `margin` rule altogether.

What do folks think? (cc @mirka @jasmussen )

Reply:
High level I do like the idea of unifying and systematizing how the spinner appears, for example always centered. But it also seems like something that needs a little exploration and guideline in terms of where the spinner is currently used, and might involve updating the places where it's used. That seems to suggest a separate exploratory PR for that, and then in order to land this big improvement, perhaps going back to inline-block. Though I do think we find a better margin, perhaps `$grid-unit-10 $grid-unit-10 0`?

Reply:
I agree with your point.

Probably best thing to do here is to switch back to the values we had before (`display: inline-block` and `margin: 5px 11px 0`), to make sure we don't introduce any breaking changes.

We can then definitely explore what to do separately, as you suggested

Reply:
I'm unaware of the history around the margin value, but the results are suboptimal in places like the Site Logo and Template Part blocks. `margin: auto;` works better as a default in that regard.

Then again the idea of having no margin on the Spinner at all does seem better on the surface. So I'm happy to leave the margin/display values as-is and revisit separately.

Reply:
Oh definitely we need that. Just in order to land this one with minimal disruption, we can pull back on those, and then follow up in small PRs, either a separate PR that just re-adds the auto margin entirely, or adds the margin in the contexts where they are needed.

Reply:
Cool, I'll go ahead, push these changes, give it a "final final" look and hopefully merge

Reply:
Merged!

> follow up in small PRs, either a separate PR that just re-adds the auto margin entirely, or adds the margin in the contexts where they are needed.

My preference would be to remove the `margin` rule altogether from `Spinner` and add any necessary `margin` where each instance of `Spinner` is used

Reply:
Generally agree that the component itself should be unopinionated and subject to its container. But given very few existing containers are likely to be aware of this new responsibility, it might still make sense to have something. We could then consider that a default margin (either a fixed clearance all around, or auto margins), which would then be easily overriden by it's container?

Reply:
> My preference would be to remove the `margin` rule altogether from `Spinner` and add any necessary `margin` where each instance of `Spinner` is used

This is my general preference, too. And if the audit shows that it's frequently used as an inline item, I'd consider giving it some padding (instead of margin) so it isn't flush with the bounding box. Basically like an icon component.